tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -615,6 +615,9 @@ def test_ended(self, test, results):
             return
         if self.timer is not None:
             self.timer.cancel()
+
+        self.browser.browser.possible_parse_tomestone(self.logger)
```

Comment:
Tests are failing because not all browsers implement this function.

Reply:
Thanks, updated